src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+  type Props = {
+    movie: Movie;
+  };
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
+        />
```

Comment:
You can make deep destrution

```suggestion
export const MovieCard: React.FC<Props> = ({ movie: {imgUrl, title, description, imdbUrl} }) => (
  <>
    <div className="card-image">
      <figure className="image is-4by3">
        <img
          data-cy="MovieImage"
          src={imgUrl}
          alt="Film logo"
        />
```